src/components/templates/Search/filterPrice.tsx

```diff
@@ -36,7 +36,11 @@ export default function FilterPrice({
 }): ReactElement {
   const navigate = useNavigate()
 
-  async function applyFilter(filterBy: string) {
+  const [priceSelections, setPriceSelections] = useState<string[]>([])
+  const [serviceSelections, setServiceSelections] = useState<string[]>([])
+  const [clearSelected, setClearSelected] = useState<boolean>(false)
```

Comment:
not sure I get why the clear action gets its own state. So we want a clear button to clear all user selection. Usually a clear button is only visible after user has selected something. Then when that button/small text link is clicked all selections should be turned off, meaning we can simply reset `priceSelections` & `serviceSelections` states so I don't see a reason to have this clear button have its own state here which is then turned on/off multiple times further down. Think all we need is that `applyClearFilter` method resetting states.

UI-layout-wise I would try like this:

<img width="615" alt="Screen Shot 2021-03-02 at 12 05 27" src="https://user-images.githubusercontent.com/90316/109639528-96594e00-7b4f-11eb-8066-9c2752c9c7ec.png">

Reply:
Yep, i agree, it doesn't make sense to be the same as the other filters. It needs to be a separate action

Reply:
Almost done that. I'm only having an issue related to the new solution: after `onClick` on the `Clear` button, all elements behave correctly except the button doesn't hide unless an action is done ( a page reload, click anywhere). Once that is fixed, I'll push the solution.

Reply:
Add another state `isClearVisible` and set that `onClick` on in a `useEffect`